### PR TITLE
[13.0][MIG]sale_quotation_number: Migration to 13.0

### PR DESCRIPTION
**Pull request by manuelregidor**
Standard migration.

### CODE REVIEW

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
That shouldn't be in "General settings" but Sales settings

Captura de pantalla 2020-09-04 a las 18 17 44

File: sale_quotation_number/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "Agile Business Group, "
+    "Qubiq, "
+    "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["sale_management"],
+    "data": ["data/data.xml", "views/sales_config.xml"],
+    "demo": [],
+    "test": [],
```
Comment (nikul-serpentcs) [inline sale_quotation_number/__manifest__.py]: @manuelregidor Please remove the key, which is not added values.

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Review by rousseldenis — APPROVED** [review]
Code review

File: sale_quotation_number/__manifest__.py (hunk 1 above)
Comment (manuelregidor) [reply sale_quotation_number/__manifest__.py]: Done

**Comment by HaraldPanten** [comment]
@pedrobaeza @rafaelbn could you merge this one, please?

**Comment by rousseldenis** [comment]
> @pedrobaeza @rafaelbn could you merge this one, please?

Normally, a PR should stay five days before merge. The OCA bot will mark as 'Ready to merge' from this date. Are you in a hurry ?

**Comment by HaraldPanten** [comment]
> > @pedrobaeza @rafaelbn could you merge this one, please?
> 
> Normally, a PR should stay five days before merge. The OCA bot will mark as 'Ready to merge' from this date. Are you in a hurry ?

A little bit, yes

**Comment by pedrobaeza** [comment]
Merging without reviewing trusting on existing reviews:

/ocabot merge nobump

**Pull request merged**